athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/data/ArrowTypeComparator.java

```diff
@@ -25,7 +25,6 @@
 import org.apache.arrow.vector.types.pojo.ArrowType;
 import org.bouncycastle.util.Arrays;
 import org.joda.time.DateTimeZone;
```

Comment:
unused joda import?

Reply:
So I decided to leave 98-101 in so that it was just doing a comparison, however, after mulling over this, I think its ok to remove.